analyses/2019_04_CoderT_Adblocker and Tracker Blocker Analysis

```diff
@@ -0,0 +1,35 @@
+What is the difference between ad blocker dataset and tracker blocker dataset.#86
+
+I found a very interesting article https://www.sba-research.org/wp-content/uploads/publications/block_me_if_you_can.pdf 
+
+Summary of Article
+•	This looks at the links between ad blocking and tracker blocking on the basis of overall privacy and security of Internet users and which tools are the most effective.
+•	It looks at the tools that internet users use to block adds and to block tracking.
+•	come from the browser extensions which differentiate between tracking and non-tracking HTTP requests.  This will block the browser extensions and allow the non-tracking HTTP requests. 
+
+Blocking Trackers
+We know that network-based blocking is done through DNS blocking where addresses are blacklisted in order to block access to certain domains.  This will include ads and this method works independently of the used application.  The DNS filtering can be used to block entire domains but not individual URIs.
+
+When analyzing the data set, in relation to #24 – How does ad blocking relate to this dataset, we are sure to find entire domains being blocked because of the use of DNS blocking, and an analysis of how to factor this into the code to build a heuristic of browser attributes should be considered.  
+
+With Blocking Trackers and Interception Proxies, third party content cannot always be reliably detected at a network level, which might skew the results when looking at ad blocking in a dataset.  
+
+Interception Proxies
+Privacy is enhanced through the use of interception proxies.  The article looks at Privoxy which has a URI-based filtering capability that can modify the content and headers of web requests allowing it to be used to remove individual cookies and block certain URIs and tracking code from the web pages.
+
+With the use of proxies, we should look to see which method of proxies are being used as this will show how fingerprinting is detected and how far it goes into tracking a user.  Because interception proxies cannot modify or intercept HTTP traffic, it is likely that we will still see the fingerprint of the user even if a proxy has been used, thus again, I think that this will be good to factor into code to build a heuristic of browser attributes.  
```

Comment:
> it is likely that we will still see the fingerprint of the user even if a proxy has been used, thus again, I think that this will be good to factor into code to build a heuristic of browser attributes.  

This is interesting. Can you expand a bit more. How would you see the fingerprint if you were capturing proxy traffic?